### DISCUSSION TITLE
ZJ Beny Charger

### DISCUSSION THREAD

**Discussion by GuzziDoug** [lbbrhzn/ocpp#690] [Q&A]
I have a ZJ Beny  BCP-A2N-P OCPP charger. (This is similar to the original Tesla charger, but different control electronics).

 I have this unit working. Everything so far seems to work.with no issues.

 This is a very reasonably priced charger, so good to see it works well.

 One query I have is about setting up the Rfid tags: is there a configuration for Rfids?
(the tag worked, but might be because details saved on charger from my ´SteVe´ server
I originally set the charger up on.)

**Comment by drc38** [comment]
Refer to #370

**Comment by GuzziDoug** [comment]
Thanks for the reply re Rfid. Searched but not for RFC! 
 I will wait for the implementation, but in my rural case, it is not an issue.
